### PR TITLE
Add Puluminaries page

### PR DESCRIPTION
**Pull request by mattstratton**
Adds a page listing all the Puluminaries. This is based upon the `data/team` directory, etc.

Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

### CODE REVIEW

**Comment by mattstratton** [comment]
I'm thinking of removing the title/company for each person, because not everyone has that set, and also that is something that changes frequently enough to be a pain to maintain. Thoughts?

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ date: 2022-05-03T10:04:47-05:00
 # Use the meta_desc property to provide a brief summary (one or two sentences)
 # of the content of the post, which is useful for targeting search results or social-media
 # previews. This field is required or the build will fail the linter test.
-meta_desc: We are very excited to announce and showcase our new program of community champions, the Puluminaries! 
+meta_desc: We are very excited to announce and showcase our new program of community champions, the Puluminaries!
```
Comment (aaronkao) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:13]: that shouldn't be the meta description which is saying its an announcement

File: themes/default/content/community/puluminaries/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Puluminaries
+meta_desc: The Developer Advocate Team at Pulumi is here to help you understand cloud engineering to build, deploy, and manage your whole stack. Use this page to reach us.
```
Comment (aaronkao) [inline themes/default/content/community/puluminaries/_index.md:3]: meta seems wrong

File: themes/default/layouts/community/puluminaries/list.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{{ define "hero" }}
+    {{ partial "hero" (dict "title" "Puluminaries") }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="max-w-4xl container mx-auto py-8 px-4 text-center text-lg">
+        <p>
+            The Puluminaries program was created to recognize members of the Pulumi community who are experts and leaders in the field of programming and infrastructure, but also
```
Comment (aaronkao) [inline themes/default/layouts/community/puluminaries/list.html:8]: this should not be past tense. it should state what it "is"

File: themes/default/content/blog/introducing-the-puluminaries/index.md (hunk 1 above)
Comment (mattstratton) [reply themes/default/content/blog/introducing-the-puluminaries/index.md:13]: This file is the existing blog post - the only change was in the redirect.

File: themes/default/content/community/puluminaries/_index.md (hunk 2 above)
Comment (mattstratton) [reply themes/default/content/community/puluminaries/_index.md:3]: Oh whoops. You're correct. I will fix.

File: themes/default/data/team/team/simen-a-w-olsen.toml [hunk 4]
```diff
@@ -2,6 +2,7 @@ id = "simen-a-w-olsen"
 name = "Simen A. W. Olsen"
 title = "Software Architect and Manager"
```
[the remark this answers is not in the document]
Comment (mattstratton) [reply themes/default/data/team/team/simen-a-w-olsen.toml:3]: I made a comment earlier in the PR asking about this - we need to decide if we want them at all.

File: themes/default/content/blog/introducing-the-puluminaries/index.md (hunk 1 above)
Comment (aaronkao) [reply themes/default/content/blog/introducing-the-puluminaries/index.md:13]: ah, got it

File: themes/default/content/community/puluminaries/_index.md (hunk 2 above)
Comment (mattstratton) [reply themes/default/content/community/puluminaries/_index.md]: Fixed in 29eb231810ac9059c36a337d3464470a2f3f1785

File: themes/default/data/team/team/simen-a-w-olsen.toml (hunk 4 above)
[the remark this answers is not in the document]
Comment (mattstratton) [reply themes/default/data/team/team/simen-a-w-olsen.toml:3]: Fixed in 9ff3193a61ebf851d6d5d18c786d6409e8ca2ede

File: themes/default/layouts/community/puluminaries/list.html (hunk 3 above)
Comment (mattstratton) [reply themes/default/layouts/community/puluminaries/list.html]: Fixed in 1d46ec204a62f8697284257f2c3d78c43e99f982

**Review by cnunciato — CHANGES_REQUESTED** [review]
Content and layout look good, but we should fix up some of these images -- Engin's has a seam along the top edge, Kat's is a bit blurry and overclose compared to the others, Paul's is being upscaled, and Alexandre's and Joshua's are too large file-size-wise.

**Review by susanev — COMMENTED** [review]
its essential that we think about the holistic experience when adding new pages.
how do people get to this page in our navigation structure?
should we, at a minimum, add a link from the community page?
what other pages have visitors that may be interested in learning more about the pulumianries?

**Comment by mattstratton** [comment]
@susanev 
Right now, there isn't anything in the navigation structure, I see this as a landing page (today pulumi.com/puluminaries points to a blog post announcing the program, this replaces that url). 

I know that @katcosgrove is working on the Community page, which I assume would include a link to the Puluminaries page.

**Comment by susanev** [comment]
@mattstratton no need to block this particular pr, but we really need to figure out a navigation structure that works for marketing needs, because there are soo many pages that exist totally outside our navigation structure and this is becoming a usability issue. if the current navigation structure doesn't work for yall we need to figure out one that does.

**Review by cnunciato — APPROVED** [review]
:shipit:

File: themes/default/data/team/team/joshua-studt.toml [hunk 5]
```diff
@@ -2,6 +2,7 @@ id = "joshua-studt"
 name = "Joshua Studt"
 title = "Solutions Architect, Financial Independence Group"
```
Comment (aaronkao) [inline themes/default/data/team/team/joshua-studt.toml:3]: we should break this into title and company like the rest of them

Comment (mattstratton) [reply themes/default/data/team/team/joshua-studt.toml:3]: As this has been merged, I opened a new PR to fix 

https://github.com/pulumi/pulumi-hugo/pull/1641

**Pull request merged**